buildscripts/cross.sh

```diff
@@ -15,19 +20,23 @@ for os in "$@"; do
 	if [[ "${GOOS}" == "darwin" ]]; then
 		export CC="o64-clang"
 		export CXX="o64-clang++"
-		# -ldflags=-s:  see https://github.com/golang/go/issues/11994
-		export LDFLAGS="${GO_LDFLAGS} -ldflags=-s"
+		# -ldflags=-s:  see https://github.com/golang/go/issues/11994 - TODO: this has been fixed in go 1.7.1
```

Comment:
Not sure if I understand if this is fixed by #943?  Do we still need `LDFLAGS="-s"`?